scaffold/pantheon/pantheon.yml

```diff
@@ -2,6 +2,6 @@ api_version: 1
 web_docroot: true
 php_version: 8.0
 database:
-  version: 10.4
+  version: 10.11
```

Comment:
How do we feel about this change going out to all sites? Does this upgrade to several versions work okay on Pantheon deployments?

Reply:
I feel like I don't see 10.11 listed on https://docs.pantheon.io/pantheon-yml#specify-a-version-of-mariadb so does this need to be 10.6?